### ISSUE TITLE
GenotypeGVCFs returns a fatal error.

### ISSUE THREAD

**Issue by aliveben2k** [broadinstitute/gatk#7008]
## Bug Report
When I use output files of CombineGVCFs to run GenotypeGVCFs, it seems no problem at beginning. However,  several hours later, it suddenly shot down. The fatal error occur.

### Affected tool(s) or class(es)
GenotypeGVCFs, only use arguments: -R, -V, -O, -all-sites

### Affected version(s)
GATK4 v4.1.9.0

### Description 
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGBUS (0x7) at pc=0x00002acb0aee41d3, pid=14508, tid=0x00002acb0f80b700
#
# JRE version: OpenJDK Runtime Environment (8.0_152-b12) (build 1.8.0_152-release-1056-b12)
# Java VM: OpenJDK 64-Bit Server VM (25.152-b12 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x1501d3]  __memmove_ssse3_back+0x1a13
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#

---------------  T H R E A D  ---------------

Current thread (0x00002acb10021800):  GCTaskThread [stack: 0x00002acb0f70b000,0x00002acb0f80c000] [id=14511]

siginfo: si_signo: 7 (SIGBUS), si_code: 2 (BUS_ADRERR), si_addr: 0x00000003ea598000

Registers:
RAX=0x00000003ea593600, RBX=0x00002acb0f80aa00, RCX=0x00000000000059c8, RDX=0x0000000000000f48
RSP=0x00002acb0f80a928, RBP=0x00002acb0f80a950, RSI=0x000000044246f290, RDI=0x00000003ea597fa0
R8 =0x00000003ea593600, R9 =0x0000000057ed72f0, R10=0x00000003c0000000, R11=0x00002acb0af16b50
R12=0x0000000000000b3d, R13=0x00000000000059e8, R14=0x00002acb0f80aa00, R15=0x0000000010490000
RIP=0x00002acb0aee41d3, EFLAGS=0x0000000000010206, CSGSFS=0x0000000000000033, ERR=0x0000000000000006
  TRAPNO=0x000000000000000e

Top of Stack: (sp=0x00002acb0f80a928)
0x00002acb0f80a928:   00002acb0ba575c6 00000003c5a14ae8
0x00002acb0f80a938:   0000000000412400 000000001048e05a
0x00002acb0f80a948:   00002acb0c077d00 00002acb0f80a9a0
0x00002acb0f80a958:   00002acb0ba155e8 00002acb0c03f148
0x00002acb0f80a968:   0000000010490000 00000003c5a14ae8
0x00002acb0f80a978:   0000000442480000 0000000000001048
0x00002acb0f80a988:   00002acb0f80aa00 0000000442400000
0x00002acb0f80a998:   00002acb0c03f148 00002acb0f80aa80
0x00002acb0f80a9a8:   00002acb0ba5ad11 00002acb0a767c53
0x00002acb0f80a9b8:   3f002acb2735a408 00000004000103b0
0x00002acb0f80a9c8:   0000000000003810 00002acb0f8193b8
0x00002acb0f80a9d8:   00002acb0f80aa4c 00002acb0f80aa40
0x00002acb0f80a9e8:   0000000000411bf0 000000000000b20c
0x00002acb0f80a9f8:   00002acb100b8550 00002acb0c02dfa0
0x00002acb0f80aa08:   00002acb0c077d00 00002acb100b8550
0x00002acb0f80aa18:   000000000000d940 000000044246a8f0
0x00002acb0f80aa28:   00002acb1001b410 00000003ea593600
0x00002acb0f80aa38:   00002acb10022800 00000005c4c6bdc8
0x00002acb0f80aa48:   000000000c073d60 00002acb0f80aaa0
0x00002acb0f80aa58:   00002acb10098240 00002acb100b8550
0x00002acb0f80aa68:   00002acb0f80aaa0 00002acb0f80aab8
0x00002acb0f80aa78:   0000000000000000 00002acb0f80aaf0
0x00002acb0f80aa88:   00002acb0ba4f261 0000000000000000
0x00002acb0f80aa98:   00002acb0c073d60 00002acb0f80aaf0
0x00002acb0f80aaa8:   00002acb0b9c17ae 00002acb1001f280
0x00002acb0f80aab8:   000000000000054b 0000000000000000
0x00002acb0f80aac8:   00002acb100b8550 0000000000000001
0x00002acb0f80aad8:   00002acb1001f201 0000000000000001
0x00002acb0f80aae8:   00002acb0c06dd72 00002acb0f80ab30
0x00002acb0f80aaf8:   00002acb0ba36edc 00002acb0f80ab30
0x00002acb0f80ab08:   00002acb10021800 00002acb0c06de6d
0x00002acb0f80ab18:   0000000000000000 00002acb5005e800 

Instructions: (pc=0x00002acb0aee41d3)
0x00002acb0aee41b3:   6f 7e 70 66 0f 7f 07 66 0f 7f 4f 10 66 0f 7f 57
0x00002acb0aee41c3:   20 66 0f 7f 5f 30 66 0f 7f 67 40 66 0f 7f 6f 50
0x00002acb0aee41d3:   66 0f 7f 77 60 66 0f 7f 7f 70 48 8d b6 80 00 00
0x00002acb0aee41e3:   00 48 8d bf 80 00 00 00 0f 83 7b ff ff ff 48 81 

Register to memory mapping:

RAX=0x00000003ea593600 is an oop
[C 
 - klass: {type array char}
 - length: 11499
RBX=0x00002acb0f80aa00 is an unknown value
RCX=0x00000000000059c8 is an unknown value
RDX=0x0000000000000f48 is an unknown value
RSP=0x00002acb0f80a928 is an unknown value
RBP=0x00002acb0f80a950 is an unknown value
RSI=(base)

#### Steps to reproduce
First step I used:
gatk --java-options "-Xmx16g -XX:ParallelGCThreads=2" CombineGVCFs -R REF.fasta -L Chr01 --variant 001.g.vcf.gz --variant 002.g.vcf.gz --variant 003.g.vcf.gz -O combined_vcf_Chr01.g.vcf
Second step I used:
gatk --java-options "-Xmx16g -XX:ParallelGCThreads=2" GenotypeGVCFs -R REF.fasta -V combined_vcf_Chr01.g.vcf -O output_Chr01.vcf -all-sites

**Comment by droazen** [comment]
@aliveben2k Are you still encountering this error?

**Comment by aliveben2k** [comment]
Yes, this happened repeatedly. Now I use GenomicsDBImport instead of CombineGVCFs, and it seems no problem. However, it is much slower than CombineGVCFs when import samples into GenomicsDB even using batch and consolidate arguments in our server.

**Comment by aliveben2k** [comment]
BTW, if I use output files of CombineGVCFs as input files of GenotypeGVCFs, and split chromosome sequence into small pieces (1,000,000 bp) such as:
gatk --java-options "-Xmx16g -XX:ParallelGCThreads=2" GenotypeGVCFs -R REF.fasta -V combined_vcf_Chr01.g.vcf -O output_Chr01_1.vcf **-L Chr01:1-1000000** -all-sites
if there is no variant site, the output file will only contain header lines. No non-variant site will present in the vcf file. However, I used **-all-sites** argument. Is it a bug?
